controllers/OrderController.js

```diff
@@ -0,0 +1,44 @@
+import { OrderService } from '../services';
+
+const findCartItems = async (req, res) => {
+  try {
+    const { id } = req.foundUser[0];
+    const cartItems = await OrderService.findCartItems(id);
+
```

Comment:
```suggestion
    const { id } = req.foundUser[0];
    const cartItems = await OrderService.findCartItems(userId);

```
백엔드에서 어떤 `id`를 주고받을 때에는 이것이 보통 테이블이의 `id`이기 때문에
어떤 테이블의 `id`인지 헷갈리기가 쉽습니다.
따라서, 수정하려는 테이블의 자기 자신의 id라면 상관 없지만, 지금과 같이 `findCartItems`라는 함수에게 인자로 `id`를 넘겨준다면, 이는 다른 개발자로 하여금 "아 카트 아이템 테이블의 아이디구나" 하고 오해를 불러일으킬 확률이 높습니다 :)
물론 users테이블에 관련된 함수라면 그냥 `id`여도 "아 유저 테이블의 id구나"하고 오해하지 않겠지만요

따라서 현재 이 controller, service,dao에 있는 `id`라는 변수가 모두 `userId`로 바뀌는 것이 코드의 유지보수를 위한 가독성에 훨씬 도움이 될 것 같습니다 하영님 👍

Reply:
앗 너무 감사합니다 소헌님!!!
그렇군요 테이블이 많으니 각종 `id`가 존재할 것이고
이것이 어떤 `id`인지 명확하게 알 수 있게끔 해주는 것이 좋겠네요!
감사합니다. 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-backend/pull/8/commits/43c97e8d880e6214613f802ef866ea62b583cb60

Reply:
userId 수정이 누락된 부분을 추가로 수정했습니다!!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-backend/pull/8/commits/7388a329d58881b197a5c63acf48f83f17e7ec11